CS.Security/Interfaces/IUserService.cs

```diff
@@ -1,14 +1,14 @@
 using CS.Security.DTO;
 
-namespace CS.Security.Interfaces
+namespace CS.Security.Interfaces;
+
+public interface IUserService
 {
-    public interface IUserService
-    {
-        Task<UserDto> Create(UserSignUpDto user);
-        Task<TokenDto> VerifyToken(TokenDto token);
-        Task<TokenDto> GetTokens(UserLogInDto user);
-        Task<bool> VerifyEmail(Guid userId, string code);
-        Task<bool> Delete(string userId);
-        Task<UserInfoDto> GetById(Guid userId);
-    }
-}
+    Task<UserDto> Create(UserSignUpDto user, string role, bool sendInvitationEmail);
+    Task<TokenDto> VerifyToken(TokenDto token);
+    Task<TokenDto> GetTokens(UserLogInDto user);
+    Task<bool> VerifyEmail(Guid userId, string code);
+    Task<bool> Delete(string userId);
+    Task<UserInfoDto> GetById(Guid userId);
+    Task<List<UserInfoDto>> GetAll(List<string> roleNames);
```

Comment:
One method that was specific to Admin service was getting all admins. But we can think few steps ahead and create more generic method that returns users of specific roles, which we can use to get admins only then.